hugo/assets/scss/components/article.scss

```diff
@@ -22,6 +22,24 @@
         position: relative;
     }
 
+    &__footer {
+        border-top: solid 2px $c-yellow;
+        display: flex;
+        flex-direction: column;
+        gap: 2rem;
+        margin: 0 auto;
+        opacity: 1;
+        padding: 24px 0;
+    }
+
+    &__info {
+        column-gap: 1rem;
+        display: flex;
+        flex-wrap: wrap;
+        margin: 1.5rem 0;
```

Comment:
The margin is too big. The heading should be compact, especially since there's a lot of info there (date, reading time, title, authors, tags etc). Also visual the distance to the title is now too big which makes the tags 'float' (not belong anywhere..). Maybe use 0.5 rem?
![Screenshot from 2023-08-07 10-28-50](https://github.com/cue-lang/cuelang.org/assets/122357989/8e8ec769-0237-480b-9f31-8594343b3391)